example/standalone.js

```diff
@@ -1,6 +1,6 @@
 const { OpenApi } = require('../dist')
 
-OpenApi.init('./petstore.json', 'http://petstore.swagger.io/v2').then(binding => {
+OpenApi.init('./petstore.json').then(binding => {
```

Comment:
The hostname doesn't have to be passed anymore since it can be read from the schema itself.